src/Features/Core/Portable/Snippets/RoslynLSPSnippetItem.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Snippets
+{
+    internal readonly struct RoslynLSPSnippetItem
```

Comment:
doc.

Reply:
`Item` makes it sound like it's an entire snippet. I'd suggest renaming `Item` => `Part` to indicate it's just a piece